docs/source/documentation/reconstruction/from_raw_data_to_tractogram.rst

```diff
@@ -0,0 +1,127 @@
+
+This should produce a file named `b0_mean.nii.gz` which is the average of all b0 images. It should look like this:
+
+.. image:: ../../_static/b0_mean.png
+   :scale: 20%
+
+(See :ref:`Create overlapping slice mosaics<create_overlapping_slice_mosaics>` for instructions on how to create such a screenshot.)
+
+Preprocessing of the DWI can include rejecting problematic volumes with :ref:`scil_dwi_detect_volume_outliers` or denoising with :ref:`scil_denoising_nlmeans`. The user may include other preprocessing steps of their choice, such as registration to the MNI template with ANTS or skull-stripping with FSL.
+
+Reconstruction
+----------------
+
+scilpy has scripts to compute various local reconstruction methods (:ref:`scil_dti_metrics`, :ref:`scil_dki_metrics`, :ref:`scil_qball_metrics`, :ref:`scil_freewater_maps`, :ref:`scil_NODDI_maps`), particularly many variations of fODFs (:ref:`scil_fodf_metrics`, :ref:`scil_fodf_msmt`, :ref:`scil_fodf_memsmt`, :ref:`scil_fodf_ssst`). 
+
+Let's start with diffusion tensor imaging (DTI).
+
+DTI reconstruction 
+----------------
+
+We will extract the shell with a b-value around 1000 s/mm² for DTI fitting using :ref:`scil_dwi_extract_shell`. We will also include the b0 images.
+::
+
+    scil_dwi_extract_shell dwi.nii.gz dwi.bval dwi.bvec 0 1000 dwi_b1000.nii.gz dwi_b1000.bval dwi_b1000.bvec
+
+Now, we can reconstruct the DTI tensor volume using :ref:`scil_dti_metrics`. scilpy automatically computes the FA, MD, RGB, and eigenvectors maps in addition to the tensor volume. To ease computation, we will constrain the fitting to the brain mask.
+::
+
+    scil_dti_metrics dwi_b1000.nii.gz dwi_b1000.bval dwi_b1000.bvec --mask brainmask.nii.gz
+
+This will produce a lot of files, including:
+
+* `tensor.nii.gz`: The diffusion tensor volume
+* `fa.nii.gz`: Fractional Anisotropy map
+* `md.nii.gz`: Mean Diffusivity map
+* `rd.nii.gz`: Radial Diffusivity map
+* `rgb.nii.gz`: RGB map of the principal diffusion direction
+* `tensor_evecs.nii.gz`: Eigenvectors of the diffusion tensor
+* `tensor_evals.nii.gz`: Eigenvalues of the diffusion tensor
+
+See :ref:`scil_dti_metrics` for a full list of outputs and more details.
+
+Here is an example FA map:
+
+.. image:: ../../_static/fa.png
+   :scale: 20%
+
+and an RGB map:
+
+.. image:: ../../_static/rgb.png
+   :scale: 20%
+
+
+DTI Tractography
+------------------
+
+Finally, we can do some basic deterministic tractography using the principal eigenvector (`tensor_evecs_v1.nii.gz`) of the DTI. We will use :ref:`scil_tracking_local` with the [EUDX]_ algorithm. We will seed from the white matter mask and constrain tracking to stay within it. We will generate 20,000 seeds and only keep streamline with lengths between 20 and 200 mm. We will also apply a compression factor of 0.1 to reduce file size.
+::
+
+    scil_tracking_local tensor_evecs_v1.nii.gz wm_mask.nii.gz wm_mask.nii.gz tractogram.trk --algo eudx --nt 20000 --min_length 20 --max_length 200 --compress 0.1
```

Comment:
Could you please add some `\` to place scripts on multiple line to have a smaller width of text?